apps/extension/src/routes/page/onboarding/set-password.tsx

```diff
@@ -16,16 +16,16 @@ import { PasswordInput } from '../../../shared/components/password-input';
 
 export const SetPassword = () => {
   const navigate = usePageNav();
-  const finalOnboardingSave = useOnboardingSave();
+  const onboardingSave = useOnboardingSave();
   const [password, setPassword] = useState('');
   const [confirmation, setConfirmation] = useState('');
 
   const handleSubmit = (event: FormEvent | MouseEvent) => {
     event.preventDefault();
 
     void (async () => {
-      await finalOnboardingSave(password);
-      navigate(PagePath.ONBOARDING_SUCCESS);
+      await onboardingSave(password);
+      navigate(PagePath.SET_RPC_ENDPOINT);
```

Comment:
Note that I didn't make this change in the `RestorePassword` component. It doesn't seem to be part of the onboarding flow, and I actually am unclear on if that component is actually used anywhere?

Reply:
Feel free to delete it. It's unused for like 8 months.